diff -urNp -x '*.orig' CUnit-2.1-3.org/CUnit/Sources/Makefile.am CUnit-2.1-3/CUnit/Sources/Makefile.am
--- CUnit-2.1-3.org/CUnit/Sources/Makefile.am	2014-04-24 02:54:45.000000000 +0200
+++ CUnit-2.1-3/CUnit/Sources/Makefile.am	2023-07-23 15:06:33.699436526 +0200
@@ -69,6 +69,9 @@ libcunit_la_SOURCES =
 libcunit_la_LIBADD = $(OBJECT_FILES_SHARED)
 libcunit_la_LDFLAGS = -version-info @LIBTOOL_CURRENT@:@LIBTOOL_REVISION@:@LIBTOOL_AGE@
 
+if ENABLE_CURSES
+libcunit_la_LIBADD += -l$(CURSES_LIB)
+endif
 
 if ENABLE_TEST
 noinst_LIBRARIES = libcunit_test.a
diff -urNp -x '*.orig' CUnit-2.1-3.org/CUnit/Sources/Test/Makefile.am CUnit-2.1-3/CUnit/Sources/Test/Makefile.am
--- CUnit-2.1-3.org/CUnit/Sources/Test/Makefile.am	2014-04-24 02:54:45.000000000 +0200
+++ CUnit-2.1-3/CUnit/Sources/Test/Makefile.am	2023-07-23 15:06:33.698436512 +0200
@@ -6,8 +6,7 @@ testdir=${datadir}/@PACKAGE@/Test
 
 test_PROGRAMS = test_cunit
 test_cunit_SOURCES = test_cunit.c
-test_cunit_LDADD = ../libcunit_test.a
-test_cunit_LDFLAGS = -L..
+test_cunit_LDADD = $(top_builddir)/CUnit/Sources/libcunit_test.a
 
 INCLUDES = -DMEMTRACE -DCUNIT_BUILD_TESTS -DCUNIT_DO_NOT_DEFINE_UNLESS_BUILDING_TESTS
 endif
diff -urNp -x '*.orig' CUnit-2.1-3.org/Examples/AutomatedTest/Makefile.am CUnit-2.1-3/Examples/AutomatedTest/Makefile.am
--- CUnit-2.1-3.org/Examples/AutomatedTest/Makefile.am	2014-04-24 02:54:45.000000000 +0200
+++ CUnit-2.1-3/Examples/AutomatedTest/Makefile.am	2023-07-23 15:06:33.698436512 +0200
@@ -8,8 +8,8 @@ if ENABLE_AUTOMATED
 example_PROGRAMS = AutomatedTest
 
 AutomatedTest_SOURCES = AutomatedTest.c
-AutomatedTest_LDADD = -lcunit -lcunitexamples
-AutomatedTest_LDFLAGS = -L.. -lm
+AutomatedTest_LDADD = $(top_builddir)/CUnit/Sources/libcunit.la $(top_builddir)/Examples/libcunitexamples.a
+AutomatedTest_LDFLAGS = -lm
 INCLUDES = -I..
 
 if ENABLE_CURSES
diff -urNp -x '*.orig' CUnit-2.1-3.org/Examples/BasicTest/Makefile.am CUnit-2.1-3/Examples/BasicTest/Makefile.am
--- CUnit-2.1-3.org/Examples/BasicTest/Makefile.am	2014-04-24 02:54:45.000000000 +0200
+++ CUnit-2.1-3/Examples/BasicTest/Makefile.am	2023-07-23 15:06:33.698436512 +0200
@@ -8,8 +8,8 @@ if ENABLE_BASIC
 example_PROGRAMS = BasicTest
 
 BasicTest_SOURCES = BasicTest.c
-BasicTest_LDADD = -lcunit -lcunitexamples
-BasicTest_LDFLAGS = -L.. -lm
+BasicTest_LDADD = $(top_builddir)/CUnit/Sources/libcunit.la $(top_builddir)/Examples/libcunitexamples.a
+BasicTest_LDFLAGS = -lm
 INCLUDES = -I..
 
 if ENABLE_CURSES
diff -urNp -x '*.orig' CUnit-2.1-3.org/Examples/ConsoleTest/Makefile.am CUnit-2.1-3/Examples/ConsoleTest/Makefile.am
--- CUnit-2.1-3.org/Examples/ConsoleTest/Makefile.am	2014-04-24 02:54:45.000000000 +0200
+++ CUnit-2.1-3/Examples/ConsoleTest/Makefile.am	2023-07-23 15:06:33.698436512 +0200
@@ -8,8 +8,8 @@ if ENABLE_CONSOLE
 example_PROGRAMS = ConsoleTest
 
 ConsoleTest_SOURCES = ConsoleTest.c
-ConsoleTest_LDADD = -lcunit -lcunitexamples
-ConsoleTest_LDFLAGS = -L.. -lm
+ConsoleTest_LDADD = $(top_builddir)/CUnit/Sources/libcunit.la $(top_builddir)/Examples/libcunitexamples.a
+ConsoleTest_LDFLAGS = -lm
 INCLUDES = -I..
 
 if ENABLE_CURSES
diff -urNp -x '*.orig' CUnit-2.1-3.org/Examples/CursesTest/Makefile.am CUnit-2.1-3/Examples/CursesTest/Makefile.am
--- CUnit-2.1-3.org/Examples/CursesTest/Makefile.am	2014-04-24 02:54:45.000000000 +0200
+++ CUnit-2.1-3/Examples/CursesTest/Makefile.am	2023-07-23 15:06:33.698436512 +0200
@@ -8,8 +8,8 @@ if ENABLE_CURSES
 example_PROGRAMS = CursesTest
 
 CursesTest_SOURCES = CursesTest.c
-CursesTest_LDADD = -lcunit -lcunitexamples
-CursesTest_LDFLAGS = -L.. -lm
+CursesTest_LDADD = $(top_builddir)/CUnit/Sources/libcunit.la $(top_builddir)/Examples/libcunitexamples.a
+CursesTest_LDFLAGS = -lm
 INCLUDES = -I..
 
 if ENABLE_CURSES
